### DISCUSSION TITLE
11C2C6DE-E4D9-847A-46FA-09578EE7A4B1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#154] [General]
# WASP-12

WASP-12 Ab is a hot Jupiter discovered in 2008 orbiting the star WASP-12 A, 1410 light-years away from Earth in the constellation Auriga. Due to its close proximity to its parent star, WASP-12 Ab is estimated to have an oval shape due to the star's tidal forces.

https://celestia.mobi/addon?item=11C2C6DE-E4D9-847A-46FA-09578EE7A4B1

**Comment by Floridaemojicat3** [comment]
Fire

**Comment by Zarina-5** [comment]
It **bugs** me that the planet's long point and exaggerated blackbody representation from heating is not pointed towards the star.
